postBuild

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-jupyter labextension install @jupyter-widgets/jupyterlab-manager 
+pip install -i https://test.pypi.org/simple/ widget-periodictable
```

Comment:
Do you want to suggest to install from test-pypi?

Reply:
I just use it to test the widget-periodictable package. I have released a new version at PyPi. I will change it to PyPi.